src/import_export/univcsvdialog.cpp

```diff
@@ -932,8 +934,8 @@ void mmUnivCSVDialog::OnExport(wxCommandEvent& /*event*/)
         double value = Model_Checking::balance(pBankTransaction, fromAccountID);
         account_balance += value;
 
-        const wxString& amount = Model_Currency::toString(value, currency);
-        const wxString& amount_abs = Model_Currency::toString(fabs(value), currency);
+        const wxString& amount = Model_Currency::toStringNoFormatting(value, currency);
```

Comment:
That is actually what I thought of doing at first but that will not take the precision from "currency".
I can use wxString::Format is you think using the precision is not important.
In any case I would not use "%ld" because I would want decimal point accuracy.